README.md

```diff
@@ -48,4 +48,8 @@ Give a ⭐️ if you like this project!
 ## Acknowledgments
 
 - Hat tip to Microverse
+- Background pattern [subtlepatterns](www.subtlepatterns.com)
```

Comment:
am truly sorry, the submission is a bit confusing to me. am I to finish all the projects before I submit them? because there is a review on day one and the rest days.

Please can you explain the submit process. on the capstone. what do I do about each request review the daily task

Reply:
I understand your confusion. I think it happened to me too when I first read it during my time. It took a re-read before I could grab it.

This is the point: the capstone specifies milestones each day. That does not necessarily mean that someone can not finish the entire project in one day(this seems not feasible and I don't even advise that).

So if you finished the entire project on Monday for example, you have a button on the first day to submit and so on.

You are only concerned with the submit button for the day you completed the entire project.

I think the project submission buttons stopped on Thursday. You can still submit even on Friday but that tells you the submission is late.

Does this make any sense?